eldritch/items.py

```diff
@@ -238,8 +239,41 @@ def DreadCurse(idx):
   return CombatSpell("Dread Curse", idx, {"magical": 9}, 2, -2, 2)
 
 
-class EnchantWeapon(CombatSpell):
+class BindMonster(CombatSpell):
+  def __init__(self, idx):
+    super().__init__(
+        "Bind Monster", idx, {"magical": float("inf"), "physical": float("inf")}, 2,
+        4, 2,
+    )
+    self.combat_round = None
+
+  def get_required_successes(self, state):  # pylint: disable=unused-argument
+    self.combat_round = state.event_stack[
+        -3
+    ]  # CombatRound[-3] > CombatChoice[-2] > CastSpell[-1]
+    return self.combat_round.monster.toughness(state, self.combat_round.character)
+
+  def get_usable_interrupt(self, event, owner, state):
+    if isinstance(event, events.CombatChoice) and isinstance(
+        event.combat_round.monster, monsters.Monster
+    ):
+      ret = super().get_usable_interrupt(event, owner, state)
+      return ret
+    return None
 
+  def get_cast_event(self, owner, state):
+    return events.Sequence(
+        [
+            events.PassCombatRound(self.combat_round),
+            events.TakeTrophy(owner, self.combat_round.monster),
```

Comment:
With a LootPinata, is there a risk that we TakeTrophy on it twice?

Reply:
Here's what I would do to make it so that TakeTrophy isn't done twice - I would actually change PassCombatRound to include the TakeTrophy and overwhelming damage, and then update CombatRound to just stick a PassCombatRound on the stack. (And of course change the cast event to just be PassCombatRound + Discard.)

I also recommend against saving combat_round as a property on the spell, mostly because it may lead to other subtle bugs if something changes in the CastSpell event.